### PR TITLE
Replace axios with @digitalbazaar/http-client

### PR DESCRIPTION
**Pull request by mattcollier**
ky-universal for what is done so far has been a drop-in replacement.  Test suite in bedrock-edv-storage still passes with this code!

TODO: figure out streaming API.

AND, it works straight away with the stub library being used in the local test suite.

### CODE REVIEW

File: EdvClient.js [hunk 1]
```diff
@@ -159,7 +160,14 @@ export class EdvClient {
       });
       // send request
       const {httpsAgent} = this;
-      await axios.post(url, encrypted, {headers, httpsAgent});
+      console.log('UUUUUUUU', url);
```
Comment (aljones15) [inline EdvClient.js:163]: Is this a halloween themed `console.log` message? Better to be prepared =)

**Comment by mattcollier** [comment]
I mentioned the other day that I thought there were issues with sinon stubbing with `@digitalbazaar/http-client`.  Turns out I had no idea how the mocking/stubs worked in the library.  Had to reshape the requests/responses in the mocks to the new http-client API and all is well.  Sinon stubs work fine.

Depends on: https://github.com/digitalbazaar/http-client/pull/1

**Comment by mattcollier** [comment]
Just remembered, this will be on hold until the streaming API gets reworked, there are no tests for that.

**Comment by mattcollier** [comment]
@gannan08 @dlongley this is ready for review.

**Review by gannan08 — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

**Pull request merged**